packages/payment-detection/src/thegraph/client.ts

```diff
@@ -2,9 +2,8 @@
 import { CurrencyTypes } from '@requestnetwork/types';
 import { NearChains } from '@requestnetwork/currency';
 import { GraphQLClient } from 'graphql-request';
-import { Block_Height, Maybe, getSdk } from './generated/graphql';
+import { Block_Height, getSdk, Maybe } from './generated/graphql';
 import { getSdk as getNearSdk } from './generated/graphql-near';
-import { RequestConfig } from 'graphql-request/src/types';
```

Comment:
We can't do such types of imports anymore, as `graphql-request/src/types` is not an exported module.